### PR TITLE
Добавление организатора проекта

### PR DESCRIPTION
**Pull request by NikitaGoroshko7**
https://trello.com/c/vUlzfaCq/22-%D0%B4%D0%BE%D0%B1%D0%B0%D0%B2%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5-%D0%BE%D1%80%D0%B3%D0%B0%D0%BD%D0%B8%D0%B7%D0%B0%D1%82%D0%BE%D1%80%D0%B0-%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82%D0%B0

Has been added extra info about organization in the CreateProject model and in the EditProjectViewModel
Has been added a migration
Has been changed View for CreateProject
Has been added some styles in the file general.css
Has been added a function message in the file site.js
Has been changed ProjectController

### CODE REVIEW

File: CrowdFundingApplication/Views/Project/CreateProject.cshtml [hunk 1]
```diff
@@ -168,12 +168,99 @@
...
+
+                           <div class="row">
+                               <div class="col check1">
+                                    <span asp-validation-for="FullName" class="text-danger"></span>
+                               </div>
+                           </div>
+                           <div class="row mb-3">
+                                <div class="col">
+                                    <div class="input-group">
+                                        <span class="input-group-text wdt" id="basic-addon1">ФИО</span>
```
Comment (GalayM) [inline CrowdFundingApplication/Views/Project/CreateProject.cshtml]: Why id="basic-addon1" ?

File: CrowdFundingApplication/Views/Project/CreateProject.cshtml [hunk 2]
```diff
@@ -168,12 +168,99 @@
...
+        </div>
+
         <div class=" mt-3">
             <div class="row">
                 <div class="col"></div>
                 <div class="col-lg-2"></div>
-                <div class="col-lg-2">
-                    <button class="btn btn-primary btn-block justify-content-end but_continue" type="submit">Продолжить</button>
+                <div class="col-lg-4">
+                    <button class="btn btn-primary btn-block justify-content-end but_continue" onclick="message();" type="submit">Отправить на модерацию</button>
```
Comment (GalayM) [inline CrowdFundingApplication/Views/Project/CreateProject.cshtml]: Move binding onclick="message() to js code

File: CrowdFundingApplication/Views/Project/CreateProject.cshtml (hunk 1 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/Views/Project/CreateProject.cshtml]: Has been changed

File: CrowdFundingApplication/Views/Project/CreateProject.cshtml (hunk 2 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/Views/Project/CreateProject.cshtml]: Has been changed

File: CrowdFundingApplication/Views/Project/CreateProject.cshtml [hunk 3]
```diff
@@ -168,12 +168,99 @@
...
+        </div>
+
         <div class=" mt-3">
             <div class="row">
                 <div class="col"></div>
                 <div class="col-lg-2"></div>
-                <div class="col-lg-2">
-                    <button class="btn btn-primary btn-block justify-content-end but_continue" type="submit">Продолжить</button>
+                <div class="col-lg-4">
+                    <button class="btn btn-primary btn-block justify-content-end but_continue" id="id_message" type="submit">Отправить на модерацию</button>
```
Comment (GalayM) [inline CrowdFundingApplication/Views/Project/CreateProject.cshtml]: Bad naming. id_message -> submitBotton

File: CrowdFundingApplication/wwwroot/js/site.js [hunk 4]
```diff
@@ -27,4 +27,12 @@ var loadFile = function (event) {
...
 
-CKEDITOR.replace('FullDescription');
\ No newline at end of file
+CKEDITOR.replace('FullDescription');
+
+var message = function () {
+    alert("Проект был успешно создан и отправлен на модерацию! Статус проекта можно просмотреть в профиле.");
+}
+
+$("#id_message").bind("click", function () {
```
Comment (GalayM) [inline CrowdFundingApplication/wwwroot/js/site.js]: And if the validation fails, the message will still be shown?

File: CrowdFundingApplication/Views/Project/CreateProject.cshtml (hunk 3 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/Views/Project/CreateProject.cshtml]: Has been changed

File: CrowdFundingApplication/wwwroot/js/site.js (hunk 4 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/wwwroot/js/site.js]: Has been changed

**Review by GalayM — APPROVED** [review]

**Pull request merged**